modules/cli/cmd/devrunner/config/helpers.go

```diff
@@ -351,3 +364,19 @@ func GetServiceLogLevelOverrides(serviceLogLevel string) []string {
 	}
 	return serviceLogLevelOverrides
 }
+
+// IsValidDenom checks if the input string is a valid denomination.
+//
+// A valid denomination is a string that contains only letters.
+//
+// Panics if the input string is not a valid denomination.
+func IsValidDenomOrPanic(denom string) {
+	denom = strings.ToLower(denom)
+
+	for _, r := range denom {
+		if !unicode.IsLetter(r) {
+			log.Error("Error validating denomination:", denom, "Denominations must contain only letters.")
```

Comment:
You can combine this line and the next line I think with something like `log.Fatal` or `log.Panic`